config.toml

```diff
@@ -112,6 +112,10 @@ Each BSides is a **community-driven** framework for building events for and by i
 
 We're doing tickets a little differently this year. As always, BSides Atlanta 2019 is a free event that is open to all. This year we've added a Personal Sponsor option. This sponsorship helps pay for putting the conference on and gets you early access to a ticket.
 
+We also decided to release free GA tickets in waves based on feedback from all of you. Check the schedule below and get your clicking fingers ready!
+
+[Click here](https://ti.to/bsidesatl/bsidesatl2019) to get your tickets.
```

Comment:
I like the more specific calls to action like "Get your [tickets] now" instead of "Click here" I'm find either way though.

Reply:
I'm gonna leave it like this for now because links aren't obvious with our style, and I don't know how to CSS very well